resources/benchmark-runner.mjs

```diff
@@ -384,7 +397,19 @@ export class BenchmarkRunner {
         performance.mark(prepareEndLabel);
         performance.measure("runner-prepare", prepareStartLabel, prepareEndLabel);
 
-        for (const suite of this._suites) {
+        let suites = [...this._suites];
+        if (this._suiteOrderRandomNumberGenerator) {
+            // We just do a simple Fisher-Yates shuffle based on the repeated hash of the
+            // seed. This is not a high quality RNG, but it's plenty good enough.
+            for (let i = 0; i < suites.length - 1; i++) {
+                let j = i + (this._suiteOrderRandomNumberGenerator() % (suites.length - i));
+                let tmp = suites[i];
+                suites[i] = suites[j];
+                suites[j] = tmp;
+            }
+        }
```

Comment:
This should probably go into the `runner-prepare` section.

Reply:
@camillobruni can you take another look now? I think https://github.com/WebKit/Speedometer/pull/292/commits/7a965ab5dd9f9bde09778c0878c9e3b68205e684 addresses this

Reply:
thnaks.